javascript/code-challenges/challenges-06.test.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named getNames that, given an array of people objects, uses map to return an array of names reversed.
+
+For example:
+[
+{
+  name: 'lloyd',
+  age: 32,
+  shoeSize: 12
+},
+{
+  name: 'jamie',
+  age: 21,
+  shoeSize: 8
+}
+]
+
+Returns: ['dyoll', 'eimaj'];
+------------------------------------------------------------------------------------------------ */
+
+const getNames = (arr) => {
+  // Solution code here...
+  return arr.map(obj => {
+    let name = obj.name;
+    let reverse = name.split('').reverse().join('');
+    return reverse;
+  });
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function that appends ' The end.' to a string, and returns the modified string. The original source string should not be modified.
+
+------------------------------------------------------------------------------------------------ */
+
+const appendTheEnd = (str) => {
+  // Solution code here...
+  let appended = str + ' The end.';
+  return appended;
+};
```

Comment:
Really great job on the challenges!

For this challenge as well as one below,  the value can be returned immediately. We can save space this way which is great! 👍